docs/UserGuide.md

```diff
@@ -365,8 +367,8 @@ Saves changes in details of the fosterer made in the profile page.
 
 **:exclamation: Important:**<br>
 
-* This command is only available while in fosterer profile page.
-* This command works only in a profile page, which appears after executing a view command.
+* This command is only available while in fosterer profile page.<br>
+* This command works only in a profile page, which appears after executing a view command.<br>
```

Comment:
I made changes to these two lines in my pr because they sound kind of off

Reply:
can!